### DISCUSSION TITLE
Create Rulers for Mouse Over Follow

### DISCUSSION THREAD

**Discussion by ErnestDaDev** [fabricjs/fabric.js#7808] [Q&A]
Hi Guys,
Does FabricJs have anything inbuilt to handle creating a ruler dynamically on mouseover and follow? If not, can anyone direct me on how to go about this?

Check the attached gif.

![CreateRulerOnMouseFollow](https://user-images.githubusercontent.com/78008731/158242062-a1d8a9ac-d447-468b-b00c-4ec9ce6e3e8d.gif)

Thanks in advance.

**Comment by melchiar** [comment]
See http://fabricjs.com/bounding-rectangle for an example of attaching draw function to the after:render event. You can modify that draw function to draw the lines you need.

**Comment by ErnestDaDev** [comment]
> See http://fabricjs.com/bounding-rectangle for an example of attaching draw function to the after:render event. You can modify that draw function to draw the lines you need.

Thanks @melchiar Can you please share a code with how to draw the lines and make them follow the mouse pointer.

**Reply by melchiar** [reply]
what have you tried so far?

**Reply by ErnestDaDev** [reply]
Actually found a tutorial that is closer. https://www.educative.io/collection/page/10370001/5712018204000256/4730262499885056
